### DISCUSSION TITLE
Module not found after Tailwind install

### DISCUSSION THREAD

**Discussion by diceboi** [tailwindlabs/tailwindcss#10435] [Help] [answered]
Hello! I just found something I never had before.

Just started a new nextjs project (without the new app directory) and after I installed tailwindcss, following the instructions on the website, I get this message in terminal:

Module not found: Can't resolve './img/wgeron1_2.png'

Import trace for requested module:
./styles/globals.css
./pages/_app.js

https://nextjs.org/docs/messages/module-not-found

I didnt have any "img" directory currently, and I didnt know what is that picture file. Found out if I delete the line "@tailwind utilities" from the gobals.css the problem goes away, but my tailwind classes didnt work. I tried reinstalling tailwind, restarted the dev multiple times, but unfortunately the problem stays the same. Any Idea? Thank you!

**Comment by wongjn** [comment, marked as answer]
Seems like there is some reference to `./img/wgeron1_2.png` somewhere. Perhaps do a text search for `wgeron1_2.png` or at least some part of this file name. Otherwise, if no luck, please consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by diceboi** [reply]
I tried it, and it didnt seems to find any file. To be honest its a fresh next app with some components I built, and react-icons installed, so thats why its so confusing to see this error.

Here is the public repo: 

https://github.com/diceboi/nextjsproblem.git

**Reply by wongjn** [reply]
Here is the image reference causing the error: https://github.com/diceboi/nextjsproblem/blob/68993ca299e7b6dc0fbcb8cba20e55940e1d7b42/components/slider/slidertile.js#LL6C105-L6C105

**Reply by diceboi** [reply]
Oh.. Thank you very much! Didnt search on classnames.
